packages/technical-features/kubectl-versions/README.md

```diff
@@ -0,0 +1,8 @@
+# @k8slens/kubectl-versions
+
+This package contains a injectable for a JSON object between strings of the format "MAJOR.MINOR" and "MAJOR.MINOR.PATCH".
+
+This object represents the current (at time of building each release of this package)
+greatest PATCH version for each "MAJOR.MINOR" version of `kubectl`.
+This is done at compile time so that we don't have to worry about being able to access this data on user machines.
```

Comment:
Population of `versions.json` is triggered at compile time? Is there a threat that something not-committed breaks unit tests in CI, or even gets released? Can this make compilation slower? 

Maybe triggering this should be made manual instead of automatic. WDYT?

Reply:
The whole point was to make it automatic

Reply:
Actually thinking about this more...

I don't think there is a threat that it breaks unit tests in CI. We can discuss making it periodic in the future.